sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -197,18 +194,15 @@ describe("EventHub Receiver", function(): void {
       };
       await client.send(ed, partitionId);
       debug("Sent the new message after creating the receiver. We should only receive this message.");
-      const data = await breceiver.receive(10, 20);
+      const data = await client.receiveBatch(partitionId, 10, 20, {
+        eventPosition: EventPosition.fromOffset(pInfo.lastEnqueuedOffset)
```

Comment:
We are changing what is being tested here.
This should be `EventPosition.fromEnqueuedTime(pInfo.lastEnqueuedTimeUtc)`